### DISCUSSION TITLE
Odd tags in depenizen for possibly deprecated commands?

### DISCUSSION THREAD

**Discussion by Eutherin** [GriefPrevention/GriefPrevention#1253] [Q&A]
I'm building a bossbar script for GriefPrevention in Denizen, and im seeing some tags that do not have commands attached.

does a "builder" or "manager" role still exist? "builder" tag is auto applied when you /trust someone, but I can't seem to apply it on its own. I also cannot find a "manager" command to add a player.

**Comment by RoboMWM** [comment]
Idk what denizen is, but GP isn't aware of denizen - so you may have a better opportunity finding help elsewhere.
